### PR TITLE
Filter/jeevan/form

### PR DESCRIPTION
**Pull request by Jeevan1**
Filter Form added

### CODE REVIEW

File: src/components/Tables/BaseTable.jsx [hunk 1]
```diff
@@ -106,199 +105,207 @@ const Title = styled.h1`
...
+          style={{
+            display: "flex",
+            flexDirection: "column",
+            justifyContent: "center",
+            alignItems: "start",
+            gap: theme.spacing.s4,
+          }}
+        >
+          {title && <Title>{title}</Title>}
+          <div style={{ display: "flex", alignItems: "center", gap: "12px" }}>
```
Comment (RahulRimal) [inline src/components/Tables/BaseTable.jsx:172]: replace gap with theme

File: src/components/Tables/BaseTable.jsx [hunk 2]
```diff
@@ -106,199 +105,207 @@ const Title = styled.h1`
...
+              width: theme.sizing.s256,
+              display: "flex",
+              flexWrap: "wrap",
+              gap: theme.spacing.s16,
+            }}
+            anchorEl={anchorEl}
+            setAnchorEl={setAnchorEl}
+          >
+            {table.getAllColumns().map((item) => (
+              <div style={{ display: "flex", gap: "8px" }}>
```
Comment (RahulRimal) [inline src/components/Tables/BaseTable.jsx:197]: replace gap with theme

File: src/pages/LoanFilterForm.jsx
Comment (RahulRimal) [inline src/pages/LoanFilterForm.jsx:1]: Add this form inside components

File: src/pages/LoanFilterForm.jsx [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+  padding: 0 ${({ theme }) => theme.spacing.s8};
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+  gap: ${({ theme }) => theme.spacing.s4};
+  font-size: ${({ theme }) => theme.typography.fontSize.f16};
+  box-sizing: border-box;
+  label {
+    min-width: ${({ theme }) => theme.sizing.s98};
+    // text-align: right;
```
Comment (RahulRimal) [inline src/pages/LoanFilterForm.jsx:49]: remove commented code

File: src/pages/LoanFilterForm.jsx [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
```
Comment (RahulRimal) [inline src/pages/LoanFilterForm.jsx:76]: remove commented code

File: src/pages/LoanFilterForm.jsx [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+            {section.inputs.map((input, i) =>
+              input.type === "select" ? (
+                <>
+                  <OptionWrapper style={{ flexBasis: `${input.basis}%` }}>
+                    <label htmlFor="">{input.label}</label>
+                    <select
+                      sx={{
+                        background: "inherit",
+                        border: "1px solid red",
+                        height: "30px",
```
Comment (RahulRimal) [inline src/pages/LoanFilterForm.jsx:195]: use theme here

**Review by RahulRimal — APPROVED** [review]

File: src/components/Tables/BaseTable.jsx (hunk 1 above)
Comment (Jeevan1) [reply src/components/Tables/BaseTable.jsx:172]: done

File: src/components/Tables/BaseTable.jsx (hunk 2 above)
Comment (Jeevan1) [reply src/components/Tables/BaseTable.jsx:197]: done

File: src/pages/LoanFilterForm.jsx
Comment (Jeevan1) [reply src/pages/LoanFilterForm.jsx:1]: done

File: src/pages/LoanFilterForm.jsx (hunk 3 above)
Comment (Jeevan1) [reply src/pages/LoanFilterForm.jsx:49]: done

File: src/pages/LoanFilterForm.jsx (hunk 4 above)
Comment (Jeevan1) [reply src/pages/LoanFilterForm.jsx:76]: done

**Pull request merged**